qdrant_client/http/models/models.py

```diff
@@ -1981,6 +2003,9 @@ class SparseVectorParams(BaseModel, extra="forbid"):
     index: Optional["SparseIndexParams"] = Field(
         default=None, description="Custom params for index. If none - values from collection configuration are used."
     )
+    modifier: Optional["Modifier"] = Field(
```

Comment:
should it be added to `QdrantFastembed.get_fastembed_sparse_vector_params` as well?